CHANGELOG

```diff
@@ -1,4 +1,30 @@
 # Changelog
+## [0.2.5] - 2020-02-19
+### Breaking changes
+ - Deprecate positional arguments in the following methods:
+   - "libnmstate.apply()"
+   - "libnmstate.commit()"
+   - "libnmstate.rollback()"
+   - "libmstate.show()"
+
+### New Features
+ - Report missing NM plugin during validation.
+ - Support OVS Link Aggregation (LAG).
+
+### Bug Fixes
+ - Fix "AtributeError" when editing an existing OVS interface.
+ - Do not remove slaves implicitly when removing the master interface.
+ - Fix team interface port support.
+ - Prevent virtual interface removal after DHCP or IPV6-RA timeout.
+
+### Known Issues
+
+In nmstate 0.2.5 there are some known issues:
+ - Cannot activate a connection against unmanaged network interfaces.
```

Comment:
AFAIU, this is worked around somehow. @cathay4t is mentioning a race condition in the workaround but I am not sure how this manifests.

Reply:
Hm. In that case I am removing it. Thanks!